### PR TITLE
setup.py: Dont build manpages on Windows

### PR DESCRIPTION
**Pull request by nemaniarjun**
### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [x] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [x] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

They can still be bulit via
./setup.py build_manpage

Closes https://github.com/coala/coala/issues/1694

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import datetime
 import locale
 import sys
+import platform
```
Comment (jayvdb) [inline setup.py]: imports in alpha order please.

**Comment by Nosferatul** [comment]
ack ddca6f8

**Comment by jayvdb** [comment]
could you rebase this patch please.

**Comment by sils** [comment]
ah it seems that some patch was merged after your rebase, could you rebase again? Be sure to fetch the latest changes from the original coala repo first.

**Comment by sils** [comment]
@nemaniarjun leave a comment after the rebase so we merge it right ahead without anything else getting in between ok? :)

**Comment by nemaniarjun** [comment]
Done! But circle and appveyor fail.

**Comment by Mixih** [comment]
looking into these failures

**Comment by nemaniarjun** [comment]
They are from Bear.py, unrelated to this PR.

**Comment by Mixih** [comment]
Confirmed, bear.py is failing, unrelated to this issue, master is failing cov tests
See issue #3110

**Comment by Mixih** [comment]
pull request #3111 with fix is in. @nemaniarjun it should work when you rebase it. Let us know if it fails again.

**Comment by aptrishu** [comment]
ack e6bd2d3

**Comment by nemaniarjun** [comment]
Lets get this merged before I have to rebase again :P

**Comment by sils** [comment]
@rultor merge

**Pull request merged**